cmd/pow/cmd/data/data.go

```diff
@@ -0,0 +1,21 @@
+package data
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/textileio/powergate/cmd/pow/cmd/data/get"
+	"github.com/textileio/powergate/cmd/pow/cmd/data/info"
+	"github.com/textileio/powergate/cmd/pow/cmd/data/log"
+	"github.com/textileio/powergate/cmd/pow/cmd/data/replace"
+	"github.com/textileio/powergate/cmd/pow/cmd/data/stage"
+)
+
+func init() {
+	Cmd.AddCommand(get.Cmd, log.Cmd, info.Cmd, replace.Cmd, stage.Cmd)
```

Comment:
I also like how each sub package command can just be called `Cmd` and is prefixed by its package name. No more arbitrary quasi-namespacey variable names to avoid naming conflicts.